src/tools/illink/eng/ilasm.ilproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.IL">
+
+  <!-- This project exists solely to restore ilasm for use in the test
+  project. -->
```

Comment:
Follow-up: We want to use the live built ilasm/ildasm instead of a prebuilt for the tests.